MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -32,6 +34,14 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 				self?.refreshControl?.endRefreshing()
 			}
 		}
+
+		viewModel?.onErrorStateChange = { [weak self] errorMessage in
+			guard let errorMessage = errorMessage else {
+				self?.errorView?.hideMessage()
+				return
+			}
+			self?.errorView?.show(message: errorMessage)
```

Comment:
Suggestion: 

An `if/else` statement is usually easier to read/follow in such cases:

"If there's a message to show, show it. Else, hide the existing one."

```suggestion
		if let errorMessage = errorMessage {
			self?.errorView?.show(message: errorMessage)
		} else {
			self?.errorView?.hideMessage()
		}
```

Instead of "guard against not having a message. Else, hide the existing one. Show the message if it passed the guard statement."